src/CoinLockerSystem.java

```diff
@@ -1,23 +1,20 @@
 import lombok.Getter;
 
-import java.util.ArrayList;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Scanner;
+import java.util.*;
 
 @Getter
 public class CoinLockerSystem {
     private int count = 1;
     private List<Locker> smallLockers;
     private List<Locker> mediumLockers;
     private List<Locker> largeLockers;
-    private List<Locker> usingLockers;
+    private Map<Integer, Locker> usingLockers;
 
     CoinLockerSystem() {
```

Comment:
In this case, what does it mean of this instance? I don't think this system can do anything?

Reply:
If you wanted this not to be called from outside, use private or protected.
Let's also check the Builder pattern.
https://dzone.com/articles/design-patterns-the-builder-pattern

Reply:
Builder pattern applied